### ISSUE TITLE
Error  in tracking phase

### ISSUE THREAD

**Issue by cndylan** [bertinetto/siamese-fc#38]
I try to train the network but meeting an error. 
```
train: epoch 01:   1/5985:Error using vl_nnbnorm
The MOMENTS size does not match the DATA depth.

Error in dagnn.BatchNorm/backward (line 29)
        vl_nnbnorm(inputs{1}, params{1}, params{2}, derOutputs{1}, ...

Error in dagnn.Layer/backwardAdvanced (line 120)
      [derInputs, derParams] = obj.backward ...

Error in dagnn.DagNN/eval (line 117)
  obj.layers(l).block.backwardAdvanced(obj.layers(l)) ;

Error in cnn_train_dag>processEpoch (line 253)
      net.eval(inputs, params.derOutputs, 'holdOn', s < params.numSubBatches) ;

Error in cnn_train_dag (line 105)
    [net, state] = processEpoch(net, state, params, 'train') ;

Error in experiment (line 102)
    [net, stats] = cnn_train_dag(net, imdb, batch_fn, opts.train);
```

I use matconvnet-1.0-beta25. <br>
can any help provided ? <br>
Thanks

**Comment by nanzxl** [comment]
@cndylan ,hi! I come across the same problem as yours.I use matconvnet-1.0-beta24. Have you solve the problem?

**Comment by bertinetto** [comment]
Hi, unfortunately we are not able to support newer versions of Matconvnet.
Does this error arise from the code as it is or did you modify something?

**Comment by flyingmoth** [comment]
I met the same problem. Have you solved the problem?

**Comment by nanzxl** [comment]
I simply remove the line 33 in BatchNorm.m  
% obj.moments=[ ];
Hope this helps.

**Comment by moorejee** [comment]
Is this problem caused by the different versions? I want to train the model using the last version, I am afraid of the same error.

**Comment by NeverMoreLCH** [comment]
I met the same problem. Have you solved the problem?

**Comment by ZZXin** [comment]
Same problem.

**Comment by ZZXin** [comment]
> 哦，我可以解决问题，以便培训过程可以开始。但我不知道是否会出现其他一些问题。
> 
> 我在BatchNorm.m中更改了代码。在第32行，删除参数'moment'。并删除第33行。

Solved my problem, hope not affect the final training results.

**Comment by ZZXin** [comment]
> 哦，我可以解决问题，以便培训过程可以开始。但我不知道是否会出现其他一些问题。
> 
> 我在BatchNorm.m中更改了代码。在第32行，删除参数'moment'。并删除第33行。

Solved my problem, hope not affect the final training results.

> Oh, I might solve the problem so that the training process can start. But I don't know if some other questions will occur.
> 
> I change the code in BatchNorm.m. At line 32, remove the parameter 'moments'. And remove the line 33.

Solved my problem, hope not affect the final training results.

**Comment by ZZXin** [comment]
> Oh, I might solve the problem so that the training process can start. But I don't know if some other questions will occur.
> 
> I change the code in BatchNorm.m. At line 32, remove the parameter 'moments'. And remove the line 33.

Solved my problem, hope not affect the final training results.
